Interfaces/GCClassic/main.F90

```diff
@@ -328,6 +311,9 @@ PROGRAM GEOS_Chem
      CALL Error_Stop( ErrMsg, ThisLoc )
   ENDIF
 
+  ! Set a local variable for Input_Opt%LSETH2O
```

Comment:
We can remove LSETH2O and replace all instances of it with Input_Opt%LSETH2O.

Reply:
Now fixed in https://github.com/geoschem/geos-chem/pull/1605/commits/ce9b096b1d63735625f6eb1ee459d44494babd11.